### PR TITLE
Add clang 7

### PR DESCRIPTION
**Pull request by uilianries**
Add extra compiler versions

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -49,6 +49,21 @@ matrix:
       - <<: *linux
         env: CONAN_CLANG_VERSIONS=4.0 CONAN_DOCKER_IMAGE=lasote/conanclang40 CONAN_CURRENT_PAGE=2
 
+      - <<: *linux
+        env: CONAN_CLANG_VERSIONS=5.0 CONAN_DOCKER_IMAGE=lasote/conanclang50 CONAN_CURRENT_PAGE=1
```
Comment (SSE4) [inline .travis.yml]: use conanio images pls

Comment (uilianries) [reply .travis.yml]: done!

**Comment by uilianries** [comment]
OSX 10 doesn't have fortran compiler installed:

https://travis-ci.org/conan-community/conan-lapack/jobs/492629864

Is it possible to install by brew?

**Comment by SSE4** [comment]
@uilianries I think so, at least we may install fcomp/ftran

**Comment by uilianries** [comment]
lapack requires zlib:

```
ldd /home/uilian/.conan/data/lapack/3.7.1/conan/stable/package/f16aa6248c005c19f226df75e351b1b7fbea3949/lib/liblapack.so
	linux-vdso.so.1 (0x00007ffde9de3000)
	libblas.so.3 => /home/uilian/.conan/data/lapack/3.7.1/conan/stable/package/f16aa6248c005c19f226df75e351b1b7fbea3949/lib/libblas.so.3 (0x00007febfd5f1000)
	libgfortran.so.5 => /usr/lib/libgfortran.so.5 (0x00007febfd33c000)
	libm.so.6 => /usr/lib/libm.so.6 (0x00007febfd1b7000)
	libgcc_s.so.1 => /usr/lib/libgcc_s.so.1 (0x00007febfd19d000)
	libquadmath.so.0 => /usr/lib/libquadmath.so.0 (0x00007febfd15a000)
	libc.so.6 => /usr/lib/libc.so.6 (0x00007febfcf96000)
	libz.so.1 => /usr/lib/../lib/libz.so.1 (0x00007febfcd7d000)
	/usr/lib64/ld-linux-x86-64.so.2 (0x00007febfdcfc000)

```

**Pull request merged**